docs/tutorials/embedded/wine_detector.md

```diff
@@ -0,0 +1,386 @@
+
+```bash
+python
+>>> import mxnet as mx
+```
+
+*Note: If you are getting memory allocation failed errors at this point (or at any point in this tutorial) it is likely because the full MxNet library takes up a large amount of RAM when loaded. To fit this you may want to kill the GUI as well as other processes that are occupying memory*
+
+
+## Running A Pre-Trained Inception Model on The Pi
+
+We are now ready to load a pre-trained model and run inference on the Pi. We will be using a simple object recognition model trained on the ImageNet data set. The model is called batch normalized Inception network (or Inception_BN for short) and it is found in the MxNet model zoo.
+
+### Getting the Model
+
+The first step is to download, unzip, and setup the pretrained deep network model files we will be using to classify images. To do this run the following commands in your home directory:
+
+```bash
+curl --header 'Host: data.mxnet.io' --header 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:45.0) Gecko/20100101 Firefox/45.0' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Language: en-US,en;q=0.5' --header 'Referer: http://data.mxnet.io/models/imagenet/' --header 'Connection: keep-alive' 'http://data.mxnet.io/models/imagenet/inception-bn.tar.gz' -o 'inception-bn.tar.gz' -L
+
+tar -xvzf inception-bn.tar.gz
+
+mv Inception_BN-0039.params Inception_BN-0000.params
+```
+
+### Running the Model
+
+The next step is to create a python script to load the model, and run inference on local image files. To do this create a new file in your home directory called inception_predict.py and add the following code to it:
+
+
+```python
+# inception_predict.py
+
+import mxnet as mx
+import numpy as np
+import cv2, os, urllib
+from collections import namedtuple
+Batch = namedtuple('Batch', ['data'])
+
+# Load the symbols for the networks
+with open('synset.txt', 'r') as f:
+    synsets = [l.rstrip() for l in f]
+
+# Load the network parameters
+sym, arg_params, aux_params = mx.model.load_checkpoint('Inception_BN', 0)
+
+# Load the network into an MxNet module and bind the corresponding parameters
+mod = mx.mod.Module(symbol=sym, context=mx.cpu())
+mod.bind(for_training=False, data_shapes=[('data', (1,3,224,224))])
+mod.set_params(arg_params, aux_params)
+ 
+'''
+Function to run a forward pass through the model given a pointer to an image file and the loaded network
```

Comment:
I think "model and loaded network" is a bit redundant but I like the re-statement